### PR TITLE
Issue 3156

### LINKED ISSUE #3156: Fixed header row
## Desired behavior
Creating an issue following Lucas' comment here: https://github.com/twentyhq/twenty/issues/2657#issuecomment-1869440868

It would be nice if the header row remained fixed as we scrolled (when there is a large number of items)

Screenshot 2023-12-26 at 11 34 48

## Current behavior
It's not clear what the number I'm viewing refer to
Screenshot 2023-12-26 at 11 33 00

### PR DESCRIPTION
**Pull request by ghost**
Fixes: #3156 
Changed the header table components, styling to make the header row sticky

Here's the video

https://github.com/twentyhq/twenty/assets/46224156/933d1481-5a73-49a7-885d-110c5573b43e

### CODE REVIEW

File: packages/twenty-front/src/modules/auth/sign-in-up/hooks/useSignInUp.tsx [hunk 1]
```diff
@@ -102,6 +102,9 @@ export const useSignInUp = () => {
   }, [setSignInUpStep, setSignInUpMode, isMatchingLocation]);
 
   const continueWithCredentials = useCallback(() => {
+    if (!form.getValues('email')) {
```
Comment (lucasbordeau) [inline packages/twenty-front/src/modules/auth/sign-in-up/hooks/useSignInUp.tsx:105]: This shouldn't be part of this PR.

**Review by lucasbordeau — COMMENTED** [review]
Could you wait for https://github.com/twentyhq/twenty/pull/3147 to be closed, so that you can copy the technical solution for displaying a box shadow and ensure that it works well with first column fixed.

File: packages/twenty-front/src/modules/auth/sign-in-up/hooks/useSignInUp.tsx (hunk 1 above)
Comment (dotslashbit) [reply packages/twenty-front/src/modules/auth/sign-in-up/hooks/useSignInUp.tsx:105]: Ok, I'll make the changes after #3147 is closed

**Comment by lucasbordeau** [comment]
@dotslashbit The dependency PR has been merged

**Comment by dotslashbit** [comment]
@lucasbordeau Ok, I'll now do the changes accordingly

**Comment by charlesBochet** [comment]
@dotslashbit Any update on this one? :)

**Comment by dotslashbit** [comment]
I haven't been working on this, i have some other things to do currently
Please assign it to someone else

**Comment by charlesBochet** [comment]
Ok, no worries :) Closing it
